cpp/src/arrow/chunk_resolver.h

```diff
@@ -75,8 +75,16 @@ struct ARROW_EXPORT ChunkResolver {
   mutable std::atomic<int32_t> cached_chunk_;
 
  public:
+  /// \brief Initialize from an `ArrayVector`.
   explicit ChunkResolver(const ArrayVector& chunks) noexcept;
+
+  /// \brief Initialize from a vector of raw `Array` pointers.
   explicit ChunkResolver(const std::vector<const Array*>& chunks) noexcept;
```

Comment:
@pitrou has changed these constructors a bit in this unmerged PR https://github.com/apache/arrow/pull/44217

Reply:
Ah! So you suggest I remove the documentation for these to avoid a merge conflict?